usaspending_api/api_contracts/contracts/v2/disaster/agency/count.md

```diff
@@ -0,0 +1,100 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Count of Agencies receiving disaster/emergency funding [/api/v2/disaster/agency/count/]
+
+This endpoint provides the count of Agencies which received disaster/emergency funding per the requested filters.
+
+## POST
+
+This endpoint returns a count of Agencies
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes
+        + `filter` (required, Filter, fixed-type)
+
+    + Body
+
+            {
+                "filter": {
+                    "fiscal_year": 2020,
+                    "def_codes": ["L", "M", "N"]
+                }
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `count` (required, number)
+    + Body
+
+            {
+                "count": 5
+            }
+
+# Data Structures
+
+## Filter (object)
++ `def_codes` (required, array[DEFC], fixed-type)
++ `fiscal_year` (required, number)
++ `award_type_codes` (optional, array[AwardTypeCodes], fixed-type)
```

Comment:
If I understand this filter correctly, this won't be a valid request unless it's used with the `award/count` endpoint. If I do, is there value in including it here?

Reply:
I'm not sure what the question is.

Wouldn't filtering by award type codes be useful for the count on the left? The tabs would need to request the count for each "Award category"
![image](https://user-images.githubusercontent.com/22781949/84526513-acaa3b80-ac9a-11ea-8e49-ad6e15ebbf75.png)

Reply:
I guess I was confused by `Applicable only when requested 'award' spending.` To me that implies that you need to be using the `disaster/award` endpoint

Reply:
Gotcha, yeah that comment is from a copy/paste and incorrect. Good catch